src/ontology/pato-edit.obo

```diff
@@ -10238,6 +10239,7 @@ def: "A quality of a process inhering in a bearer by virtue of the bearer's lack
 subset: relational_slim
 subset: value_slim
 is_a: PATO:0001564 ! extra or missing processual parts
+relationship: reciprocal_of PATO:0000462 ! absent
```

Comment:
this seems wrong to me.

Reply:
no. sorry. correct. it is just the way github shows these changes that made it look wrong.

Reply:
Okay, yes I see what you mean.